src/commands/bot.rs

```diff
@@ -0,0 +1,24 @@
+use crate::Context;
+use color_eyre::eyre::Result;
+use poise::{serenity_prelude as serenity, CreateReply};
+use std::env;
+
+/// Displays information about the bot
+#[poise::command(slash_command)]
+pub async fn botinfo(ctx: Context<'_>) -> Result<()> {
+    let rev = env::var("BUILD_REV").unwrap_or_else(|_| "unknown".to_string());
```

Comment:
Pretty sure you don't need a closure here if you just do something like this
```suggestion
    let rev = env::var("BUILD_REV").unwrap_or("unknown".to_string());
```

Reply:
:o